New page: xr-overlay pseudo-class (#43690)
mdn/content: files/en-us/web/css/reference/selectors/pseudo-classes/index.md

  - : Matches an element that is currently in fullscreen mode.
- {{CSSxRef(":picture-in-picture")}}
  - : Matches an element that is currently in picture-in-picture mode.
- {{CSSXref(":xr-overlay")}}

>   - : Matches the root element of the DOM overlay when a web page is being viewed in an immersive AR or VR environment.

Reviewer: ```suggestion
  - : Matches the DOM overlay element during an immersive AR or VR session.
```
